mysql-test/main/outfile_fifo.test

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
start with:
`--source include/not_windows.inc`

Reply:
Done.  I also added an ifdef to make it so the actual code change only runs on non-Windows as well.